src/input/toggle/Toggle-test.js

```diff
@@ -0,0 +1,22 @@
+import test from 'ava';
+import React from 'react';
+import {shallow} from 'enzyme';
+import sinon from 'sinon';
+import Toggle from './Toggle';
+
+test('toggle', tt => {
+
+    const onChange = sinon.spy();
```

Comment:
Yeah I gave this a go but couldn't work out how to do this with enzyme because for some reason it found the onChange function to be always undefined, when clearly it shouldn't be and it works normally when actually using it... might need to have another look.